core/slice.go

```diff
@@ -257,6 +258,7 @@ func (sl *Slice) Append(header *types.Header, domPendingHeader *types.Header, do
 	log.Info("Appended new block", "number", block.Header().Number(), "hash", block.Hash(),
 		"uncles", len(block.Uncles()), "txs", len(block.Transactions()), "etxs", len(block.ExtTransactions()), "gas", block.GasUsed(),
 		"root", block.Root())
+	sl.subAppendFeed.Send(block.Hash())
```

Comment:
This will send a message to this feed no matter if the block is a sub append or a normal append.  Is this correct?

Reply:
Good catch. No its not. Should only send to append feed. Fixed.